esp/src/src-react/components/forms/Fields.tsx

```diff
@@ -724,7 +724,7 @@ export function createInputs(fields: Fields, onChange?: (id: string, newValue: a
                         key={fieldID}
                         type="datetime-local"
                         name={fieldID}
-                        value={field.value}
+                        defaultValue={field.value}
```

Comment:
Btw, this fixes that problem I mentioned a few days ago about the native browser datetime pickers not updating the value correctly